docs/UserGuide.md

```diff
@@ -124,9 +144,10 @@ Acceptable values for each parameter:
 * `i/NRIC`: Alphanumeric, _exactly_ 4 characters.
 * `p/CONTACT NUMBER`: Numeric, _exactly_ 8 characters.
 * `e/EMAIL`: Alphanumeric /and special characters, no white spaces allowed, standard email format.
```

Comment:
Seems like because here there was an error, so yours have to. I'll make the changes accordingly

```suggestion
* `e/EMAIL`: Alphanumeric and/or special characters, no white spaces allowed, standard email format.
```